src/commands/check.ts

```diff
@@ -5,9 +5,7 @@ import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = `Displays the users current message count. 
-	If the word 'here' is used as an argument, the message count will be 
-	sent in the same channel that the command was used in (rather than the user's dms).`;
+	description = `Displays the users current message count. Use \`/check here\` to send in the current channel`;
```

Comment:
```suggestion
	description = `Displays the user's current message count.`;
```
Same case as issue.ts. This was in Blade's PR as well but suggestion listed here for good measure